schedule/kernel/ibtest-master.yaml

```diff
@@ -2,21 +2,14 @@ name:           ibtest-master
 description:    >
```

Comment:
Please create new agama-* schedules and leave old one as they are.  SLE 15 will with us  a lot of time and we can consider enabling tests in maintenance,